File: Livro - Módulo Java/3 - Estrutura de Decisão/Exercicio.22.java
// Dado três números digitados pelo usuário, e todos diferentes, imprima o número central
import java.util.Scanner;

public class Exercicio22 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        double n1 = scanner.nextDouble();

        System.out.print("Digite o segundo número: ");
        double n2 = scanner.nextDouble();

        System.out.print("Digite o terceiro número: ");
        double n3 = scanner.nextDouble();

        if (n1 != n2 && n1 != n3 && n2 != n3) {
            double numeroCentral;

            if ((n1 > n2 && n1 < n3) || (n1 < n2 && n1 > n3)) {
                numeroCentral = n1;
            } else if ((n2 > n1 && n2 < n3) || (n2 < n1 && n2 > n3)) {
                numeroCentral = n2;
            } else {
                numeroCentral = n3;
            }

            System.out.println("O número central é: " + numeroCentral);
        } else {
            System.out.println("Por favor, digite três números diferentes.");
        }

        scanner.close();
    }
}

File: Livro - Módulo Python/3 - Estrutura de Decisão/Exercicio.22.py
#23. Dado três números digitados pelo usuário, e todos diferentes, imprima o número central.

numeroA = float(input('Digite número A: '))
numeroB = float(input('Digite número B: '))
numeroC = float(input('Digite número C: '))

if numeroA > numeroB and numeroA < numeroC or numeroA < numeroB and numeroA > numeroC:
 print('Numero A é o central')
else:
  if numeroB > numeroA and numeroB < numeroC or numeroB < numeroA and numeroB > numeroC:
    print('Numero B é o central')
  else:
    if numeroC > numeroB and numeroC < numeroA or numeroC < numeroB and numeroC > numeroA:
      print('Numero C é central')
    else:
      print('Eles são iguais')